src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -123,20 +124,6 @@ namespace DataHeatBalance {
     // type (see below--Zone%OfType):
     constexpr int StandardZone(1);
 
-    // Parameters to indicate the convection correlation being used for use with
-    // InsideConvectionAlgo and OutsideConvectionAlgo
-
-    constexpr int ASHRAESimple(1);
-    constexpr int ASHRAETARP(2);
-    constexpr int CeilingDiffuser(3); // Only valid for inside use
-    constexpr int TrombeWall(4);      // Only valid for inside use
-    constexpr int TarpHcOutside(5);   // Only valid for outside use
-    constexpr int MoWiTTHcOutside(6); // Only valid for outside use
-    constexpr int DOE2HcOutside(7);   // Only valid for outside use
-    constexpr int BLASTHcOutside(8);  // Only valid for outside use
-    constexpr int AdaptiveConvectionAlgorithm(9);
-    constexpr int ASTMC1340(10);
-
```

Comment:
Moved these from here over to the proper places in the associated HcInt and HcExt convection lists of ints.